MyProject/Window Form/ViewModel/MainWindowViewModel.cs

```diff
@@ -62,9 +101,17 @@ public string Content
         }
 
         public ICommand GetTableOrder1 { get; private set; }
-        public ICommand ShowOrder { get; private set; }
+        public ICommand GetTableOrder2 { get; private set; }
+        public ICommand GetTableOrder3 { get; private set; }
+        public ICommand GetTableOrder4 { get; private set; }
+        
+        private void ActiveTblChanged(object sender)
+        {
+            //loop through toggle buttons, switch off all except the pressed one, didn't find solution yet
```

Comment:
Solution could be to have radio buttons state controlled from this ViewModel (meaning bool properties you would bind to_ and when inside this handler, mark them all as "false".

Reply:
As well you could provide the table itself tio this method and work with it. Having so, you will be able to iterate all the tables and if iterated table is not the one, make it unselected.